docs/get-started/installation/prepare/rpi-setup.md

```diff
@@ -59,92 +63,112 @@ To install Raspberry Pi OS (formerly called Raspbian) on a microSD card from whi
 
     You should be brought back to the initial launch screen.
 
-3.  To make your Raspberry Pi easier to access in the next step, configure your Raspberry Pi's hostname, ssh credentials, and wifi.
-    Click the gear-shaped settings icon in the lower right to bring up the Advanced options menu.
+1.  Now, pick your storage medium.
+    Click `CHOOSE STORAGE`:
+
+    {{< imgproc alt="Raspberry Pi Imager window showing the main page, and the 'Choose Storage' button is selected." src="/get-started/installation/rpi-setup/imager-selected-os.png" resize="800x" declaredimensions=true >}}
+
+    You may have many devices listed, select the microSD card you intend to use in your Raspberry Pi.
+    If this page is blank and you do not have any listed, make sure your microSD card is connected to your computer correctly:
+
+    {{< imgproc alt="The storage screen is shown with a generic SD card available as an option." src="/get-started/installation/rpi-setup/imager-select-storage.png" resize="800x" declaredimensions=true >}}
+
+1.  Now, configure your Raspberry Pi for remote access.
+    Click **Next**.
+    When prompted to apply OS customization settings, select **EDIT SETTINGS**.
 
     {{< imgproc alt="Raspberry Pi Imager window showing gear-shaped settings icon is selected." src="/get-started/installation/rpi-setup/advanced-options.png" resize="800x" declaredimensions=true >}}
 
     {{% alert title="Important" color="note" %}}
 
-If you are using a non-Raspberry Pi OS, altering the Advanced options will cause the initial boot to fail.
+    If you are using a non-Raspberry Pi OS, altering the OS customization settings will cause the initial boot to fail.
```

Comment:
If you look at how this is rendering, you'll see it's rendering as code - that's because unless something is indented because it's a list or something like that, indentation is seen as indication it's code. I think if you removed the new line on 83 it might work but otherwise you need to put the contents of an alert without indents
```suggestion
If you are using a non-Raspberry Pi OS, altering the OS customization settings will cause the initial boot to fail.
```